src/ompl/util/RandomNumbers.h

```diff
@@ -43,8 +43,18 @@
 #include <boost/random/variate_generator.hpp>
 #include <cassert>
 
+#include <ompl/util/ClassForward.h>
+
+//The basic Eigen include
+#include <Eigen/Core>
```

Comment:
Ohhhh. I never thought of grouping them together first...
Thanks, I think I can make that work. :smiley: